azure-pipelines.yml

```diff
@@ -218,13 +228,15 @@ jobs:
   - job: compatibility_windows
     dependsOn:
       - check_for_release
+      - patch_bazel_windows
     timeoutInMinutes: 60
     pool:
       name: 'windows-pool'
-      demands: assignment -equals default
+      demands: assignment -equals patched-bazel
     steps:
       - template: ci/report-start.yml
       - checkout: self
+      - template: ci/patch_bazel_windows/restore.yml
```

Comment:
This also needs to go in the daily cronjob as well.

Reply:
Good catch.

Reply:
Not needed at all in the new scheme.